README.md

```diff
@@ -23,6 +23,25 @@ Then you can run
 python api.py
 ```
 
+### Database Connection
+
+By default, this code uses `tests/test.db` (which is a sqlite) for testing purposes.
```

Comment:
this problem will fix with the help of docker in the near future but maybe we should consider adding this choice for users to use either MySQL or SQLite!